src/classes/Contacts.cls

```diff
@@ -491,6 +616,135 @@ public inherited sharing class Contacts extends fflib_SObjects {
         return new Contacts(contactsInHouseholdAccounts);
     }
 
+    public List<Contact> getOneToOneContactsToUpdate() {
+        List<Contact> oneToOneContactsToUpdate = new List<Contact>();
+        for (Contact contactRecord : (List<Contact>) getRecords()) {
+            Contact oldContact = oldMap.get(contactRecord.Id);
+
+            if (needsOne2OneAccountUpdate(contactRecord, oldContact)) {
+                oneToOneContactsToUpdate.add(contactRecord);
+            }
+        }
+        return oneToOneContactsToUpdate;
+    }
+
+    public Contact oldVersionOf(Contact con) {
+        return oldMap.get(con.Id);
+    }
+
+    public List<Id> getHouseholdIdsToRename() {
+        List<Id> householdIdsToRename = new List<Id>();
+        for (Contact contactRecord : (List<Contact>) getRecords()) {
+            Contact oldContact = oldVersionOf(contactRecord);
+            if (needsHHAccountRename(contactRecord, oldContact)) {
+                householdIdsToRename.add(contactRecord.AccountId);
+            }
+        }
+        return householdIdsToRename;
+    }
+
+    public void resetDeprecatedFields() {
+        for (Contact contactRecord : (List<Contact>) getRecords()) {
+            // always clear this deprecated field to avoid confusing users.
+            contactRecord.npe01__SystemAccountProcessor__c = null;
+        }
+    }
+
+    public void keepPrivateContactsPrivate() {
+        for (Contact contactRecord : (List<Contact>) getRecords()) {
+            // for contacts marked private after they have been created, break the relationship from their account (any model)
+            if (contactRecord.AccountId != null && contactRecord.npe01__Private__c == true) {
```

Comment:
Just a small cleanup item (for the future) - we could consider dropping `== true` from this conditional.